release-content/0.16/release-notes/17096_Anamorphic_Bloom.md

```diff
@@ -1,4 +1,12 @@
-<!-- Anamorphic Bloom -->
-<!-- https://github.com/bevyengine/bevy/pull/17096 -->
+Bloom is that soft glow bleeding out from bright areas of light: in the real world, this is caused by your camera (or eyes) getting overwhelmed or an imperfect focus.
```

Comment:
Nit: bloom != to defocus blur, which is what depth of field is.

I'm not an optics expert, but my understanding is bloom is a way to convey that something is bright and saturating a sensor - it differentiates between something that is paper white, vs. something that is colorful, but is saturating the sensor. This is something that is mostly useful on SDR displays. 

Arguably, if you have a bright enough HDR display, you get bloom for free because you can produce the same physical effect - it's literally a bright light!

Reply:
Yeah, I think that's a reasonable distinction to draw when talking about the postprocessing effect (even though the physical effect is a mix).